application/helpers/update/updates/Update_607.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use Exception;
+
+/**
+ * Fix organizer link : icon and survey activated
+ * @package LimeSurvey\Helpers\Update
+ */
+
+class Update_607 extends DatabaseUpdateBase
+{
+    /**
+     * @inheritDoc
+     */
+    public function up()
+    {
+        $table = $this->db->getSchema()->getTable('{{labelsets}}');
+        if (!in_array('owner_id', $table->getColumnNames())) {
+            $this->db->createCommand()->addColumn('{{labelsets}}', 'owner_id', "integer NULL");
```

Comment:
Current labelsets, which owner they have?

Reply:
`NULL` : no owner

Reply:
Can new labelsets have no owner?
Does it make sense to have some labelesets with owner and some with no owner?
Sound error-prone and complex to me.
I would rather have old labelsets to be owned by superadmin

@olleharstedt @tiborpacalat Thoughts?

Reply:
Why it's an issue ? 

A new feature for dev are allow updating owner, but it's a new feature, not a fix.

Reply:
> A new feature for dev are allow updating owner, but it's a new feature, not a fix.

https://bugs.limesurvey.org/view.php?id=18870

Reply:
> Current labelsets, which owner they have?

Can you guess a correct owner? Or do we need it to be owner = -1 --> can be edited by anyone (who has label set permission)?

Reply:
> Can you guess a correct owner

No, 
We can set only to the 1st superadmin if exist ([remind](https://github.com/LimeSurvey/LimeSurvey/blob/2077528877c30bf03d46f99994dfc538f5baef16/application/config/config-defaults.php#L673))

> Or do we need it to be owner = -1 --> can be edited by anyone (who has label set permission)?

-1 or NULL : no real difference

Need update all permission currently and with your advice … no difference.

Reply:
Maybe that's the most backward-compatible option, to allow NULL in owner_id, and apply same permission as before to them?

Reply:
> Maybe that's the most backward-compatible option, to allow NULL in owner_id, and apply same permission as before to them?

Like it done currently then.
I set it as draft : muts review import part.

Reply:
> We can set only to the 1st superadmin if exist

I would do this. Assign it to someone (first superadmin available)